docs/src/openapi-v1.yaml

```
@@ -1052,6 +1052,96 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /pallets/{palletId}/dispatchables:
+    get:
+      tags:
+      - pallets
+      summary: Get a list of dispatchables for a pallet.
+      description: Returns a list of dispatchable item metadata for distpachable items of the
+        specified palletId.
+      parameters:
+      - name: palletId
+        in: path
+        description: 'Name or index of the pallet to read dispatchable metadata for. Note: the pallet name must match what is
+          specified in the runtime metadata.'
+        required: true
+        schema:
+          type: string
+      - name: onlyIds
+        in: query
+        description: Only return the names (IDs) of the dispatchable items instead of every dispatchable's metadata.
+        required: false
+        schema:
+          type: boolean
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                description: Pallet info and Array of dispatchableItemIds.
+                items:
+                  $ref: '#/components/schemas/PalletDispatchables'
+        "400":
+          description: invalid blockId supplied for at query param
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        "404":
+          description: could not find pallet with palletId
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  /pallets/{palletId}/dispatchables/{dispatchableItemId}:
+    get:
+      tags:
+      - pallets
+      summary: Get the value of an dispatchable item.
```

```suggestion
      summary: Get the value of a dispatchable item.
```

Note if you commit this from the suggestion you'll have to rebuild the docs.